src/libs/actions/Policy.ts

```diff
@@ -3415,6 +3415,36 @@ function enablePolicyTags(policyID: string, enabled: boolean) {
 }
 
 function enablePolicyTaxes(policyID: string, enabled: boolean) {
+    const taxRatesData: OnyxData = {
+        optimisticData: [
+            {
+                onyxMethod: Onyx.METHOD.MERGE,
+                key: `${ONYXKEYS.COLLECTION.POLICY}${policyID}`,
+                value: {
+                    taxRates: {
+                        defaultExternalID: CONST.TAXES.DEFAULT.EXTERNAL_ID,
+                        defaultValue: CONST.TAXES.DEFAULT.VALUE,
+                        foreignTaxDefault: CONST.TAXES.DEFAULT.FOREIGN_ID,
+                        name: CONST.TAXES.DEFAULT.NAME,
+                        taxes: {
+                            [CONST.TAXES.DEFAULT.EXEMPT_ID]: {
+                                name: CONST.TAXES.DEFAULT.EXEMPT_NAME,
+                                value: CONST.TAXES.DEFAULT.EXEMPT_VALUE,
+                                pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD,
+                            },
+                            [CONST.TAXES.DEFAULT.TAX_RATE_1_ID]: {
+                                name: CONST.TAXES.DEFAULT.TAX_RATE_1_NAME,
+                                value: CONST.TAXES.DEFAULT.TAX_RATE_1_VALUE,
+                                pendingAction: CONST.RED_BRICK_ROAD_PENDING_ACTION.ADD,
+                            },
```

Comment:
Let's create a const with these defaults

Reply:
@luacmartins Apologies as it looks like I introduced some confusion here. I was unaware of your earlier review comments about using individual values in CONST to generate `taxRates` object. But I am ok with that too. 
So, I need a clarification here. 
Did you mean creating a `const` object for `taxRates` in the code itself and set it to `value` or did you mean creating the entire `taxRates` object in CONST? 
The reason why I ask is that @nkdengineer has reverted the code in the latest commit to create the entire `taxRates` object in CONST